python/ml4ir/base/model/relevance_model.py

```diff
@@ -325,20 +346,22 @@ def evaluate(
         Args:
             test_dataset: an instance of tf.data.dataset
             inference_signature: If using a SavedModel for prediction, specify the inference signature
+            additional_features: Additional post processing feature functions as key value pairs
+            group_metrics_min_queries: Minimum number of queries per group to be used for group aggregate metrics
+            logs_dir: Directory to log the predictions and metrics
             logging_frequency: integer representing how often(in batches) to log status
-            metric_group_keys: list of fields to compute group based metrics on
-            save_to_file: set to True to save predictions to file like self.predict()
 
         Returns:
-            metrics and groupwise metrics as pandas DataFrames
+            metrics and groupwise metrics as pandas DataFrames and flattened metrics as a dictionary
```

Comment:
We should define here what is expected as output. Reading the code we just see: `returns None, None, dict`
Ideal we should add an example. And what of each value in the Triple means.

Reply:
I've added what each value in the triple means, right? It is (metrics, groupwise_metrics, metrics_dict)
I can add example.

Reply:
Added.